nf_core/sync.py

```diff
@@ -133,6 +136,10 @@ def sync(self):
         self.make_template_pipeline()
         self.commit_template_changes()
 
+        if not self.made_changes and self.force_pr:
+            log.info("No changes made to TEMPLATE, but PR forced")
+            self.made_changes = True
+
         # Push and make a pull request if we've been asked to
         if self.made_changes and self.make_pr:
```

Comment:
Just a suggestion, could we have `if self.made_changes and self.make_pr or self.force_pr`? 
With this we can't have the log message but seems less confusing than setting `self.made_changes` to True

Reply:
good idea, I'll add it to the if clause, but also keep the log message.